.github/pkl-workflows/updateWorkflows.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+for file in *.pkl; do
+	pkl eval "$file" -o "../workflows/$(basename "${file/.pkl/.generated.yml}")"
+done
```

Comment:
It's not necessary to use a for-loop here - Pkl can evaluate multiple input files at once and template the output filename.

This can be simplified to:
```suggestion
pkl eval *.pkl -o "../workflows/%{moduleName}.generated.yml")"
```

In which case it's simple enough it probably doesn't need to be in a script! This could go nicely in a package.json "script" shortcut. And then yes, we could make "npm run package" also run this command, so generating dist and these files are one step.

Reply:
Wow! This is super nice. Thanks for leeting me know ❤️

Reply:
Fixed here https://github.com/pkl-community/setup-pkl/pull/47/commits/ed98e005893f58126d96eb0f3e7b226fe6a79c27